File: web_app/FileStorageApp/CryptoLib/MerkleTree.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CryptoLib
{
    public class MerkleTree
    {
        public List<MTMember> HashTree { get; set; }
        public int Levels { get; set; }
        public List<int> IndexOfLevel { get; set; }

        public MerkleTree()
        {
            HashTree = new List<MTMember>();
            IndexOfLevel = new List<int>();
            Levels = 0;
        }
    }
}

File: web_app/PythonProxy/CryptoFolder/MerkleTree.py
class MerkleTree:
    def __init__(self):
        self._HashTree = []
        self._Levels = 0
        self._IndexOfLevel = []
        
    def getHashTree(self):
        return self._HashTree
    
    def getLevels(self):
        return self._Levels
    
    def getIndexOfLevel(self):
        return self._IndexOfLevel